package.json

```diff
@@ -95,7 +98,7 @@
     "eslint-plugin-react": "^7.24.0",
     "execa": "^5.0.0",
     "husky": "^7.0.1",
-    "jest": "^26.6.0",
+    "jest": "^27.2.3",
```

Comment:
This is now done in https://github.com/guardian/source/pull/995. I'll rebase once that's merged.